mapper/orm/src/main/java/org/hibernate/search/mapper/orm/mapping/impl/HibernateOrmEntityTypeMetadataContributor.java

```diff
@@ -58,6 +59,7 @@ public void contributeAdditionalMetadata(PojoAdditionalMetadataCollectorTypeNode
 		// the Hibernate ORM name internally when using Hibernate ORM features (Session.load, ...).
 		node.entityName( persistentClass.getJpaEntityName() );
 		node.secondaryEntityName( persistentClass.getEntityName() );
+		node.canPotentiallyBeJoined( !( persistentClass instanceof JoinedSubclass ) );
```

Comment:
It's a bit weird to expose this metadata in pojo-base where the concept of "join" isn't even defined.

Any reason to add this to the type metadata instead of including it in the (mapper-specific) loading strategy?

For ORM you could pass it to the loading strategy in `org.hibernate.search.mapper.orm.mapping.impl.HibernateOrmEntityLoadingBinder#createLoadingStrategy`, and simply have `org.hibernate.search.mapper.orm.loading.impl.AbstractHibernateOrmLoadingStrategy#groupingAllowed` return `false` for non-joinable types.

I think that's what I suggested earlier... but maybe there was a problem I didn't anticipate?